homeassistant/components/rmvtransport/sensor.py

```diff
@@ -104,6 +72,31 @@
     async_add_entities(sensors)
 
 
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up a RMV departure sensor based on a config entry."""
+    timeout = entry.data.get(CONF_TIMEOUT, DEFAULT_TIMEOUT)
+
+    session = async_get_clientsession(hass)
+
+    rmv_data = RMVDepartureSensor(
+        session,
+        entry.data[CONF_STATION],
+        entry.data.get(CONF_DESTINATIONS, []),
+        entry.data.get(CONF_DIRECTION),
+        entry.data.get(CONF_LINES, []),
+        entry.data.get(CONF_PRODUCTS),
+        entry.data.get(CONF_TIME_OFFSET, DEFAULT_TIME_OFFSET),
+        entry.data.get(CONF_MAX_JOURNEYS, DEFAULT_MAX_JOURNEYS),
+        entry.title,
+        timeout,
+        entry.data.get(CONF_SHOW_ON_MAP),
+    )
+    await rmv_data.async_update()
+    hass.data[DOMAIN][DATA_RMVTRANSPORT_CLIENT][entry.entry_id] = rmv_data
```

Comment:
Why do we need to store the entity? Looks like we overwriting the rmv_data from the component.